### PR TITLE
Update to rust-bitcoin v0.27 & rust-elements v0.18, add Signet support

### PR DESCRIPTION
**Pull request by shesek**
The changes in electrs can be seen here: https://github.com/Blockstream/electrs/compare/new-index...Blockstream:202108-deps-signet

### CODE REVIEW

**Review by pl838831 — APPROVED** [review]

**Review by piqjen-1xofFi-sikxyh — APPROVED** [review]

**Review by pl838831 — APPROVED** [review]

**Comment by nuybad02** [comment]
61515a717b5c936c88c071c2b191b6a70943772a

**Review by piqjen-1xofFi-sikxyh — APPROVED** [review]

**Review by tongkirati — APPROVED** [review]

**Review by tongkirati — APPROVED** [review]

**Review by piqjen-1xofFi-sikxyh — APPROVED** [review]

File: flavors/bitcoin-signet/config.env [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export SITE_TITLE='Bitcoin Signet Explorer'
+export HOME_TITLE='Bitcoin Signet Explorer'
+export NATIVE_ASSET_LABEL=rBTC
```
Comment (kallewoof) [inline flavors/bitcoin-signet/config.env:5]: We currently use `sBTC` on our side, FWIW.

Comment (shesek) [reply flavors/bitcoin-signet/config.env]: Yes, this should've been sBTC, thanks for the heads up! Fixed in https://github.com/Blockstream/esplora/pull/291/commits/51ded72777cab30a3bab127d3ec8f3742e9641dc.

**Comment by piqjen-1xofFi-sikxyh** [comment]
> The changes in electrs can be seen here: https://github.com/Blockstream/electrs/compare/new-index...Blockstream:202108-deps-signet

**Comment by shesek** [comment]
The changes here were already merged as part of the `202109-liquid-testnet` branch, closing.

**Pull request closed without merging**